Resources/Audio/Voice/Talk/Silicon/attributions.yml

```diff
@@ -0,0 +1,4 @@
+- files: ["borg.ogg", "borg_ask.ogg", "borg_exclaim.ogg"]
+  license: "CC-BY-SA-4.0"
+  copyright: "Recorded and mixed by https://github.com/MilenVolf"
+  source: "https://git.arumoon.ru/Workbench-Team/space-station-14/-/merge_requests/123"
```

Comment:
This isn't even public and I don't think that's even their license.

Reply:
Oh, yeah

Reply:
It was made by me so... What license should I put in?